opus/import/table_schemas/param_info_ranges.json

```diff
@@ -0,0 +1,1091 @@
+{
+    "obs_ring_geometry": [
+        {
+            "cat_name": "jupiter rings",
```

Comment:
All of these names (category and fields) should be capitalized the way we want them to display. We shouldn't re-capitalize them later, taking control away from the person who wrote this JSON file.